### PR TITLE
New Legends Theme

### PR DESCRIPTION
**Pull request by carsenk**
New "Legends Theme" a dark brown color RS inspired theme. @bluepartyhat @redpartyhat 💯 

image

### CODE REVIEW

File: src/app/theme/legends-theme.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "--link": "#D3B882",
+    "--hover": "#e5c075",
+    "--border": "#382418",
+    "--mborder": "#382418",
+    "--filter": "invert(98%) sepia(1%) saturate(264%) hue-rotate(181deg) brightness(116%) contrast(100%)",
+    "--unread": "#1a293f",
+    "--topbar": "#16130d",
+    "--cblue": "#238eff",
+    "--cred": "#e0245e",
+    "--cgreen": "#17BF63",
```
Comment (lazynina) [inline src/app/theme/legends-theme.ts:24]: just added a new property `--loading` which sets the color of the loading spinner. It uses the same value as `--grey` for dark and icy dark, but uses `--textalt` for light theme.

Comment (lazynina) [reply src/app/theme/legends-theme.ts:24]: I'm going to create a PR to add typing for properties shortly.

**Review by lazynina — CHANGES_REQUESTED** [review]
please add `--loading` to properties and create classes to support styling the diamond popover.  You can see example styles around line 458 in styles.scss

**Comment by carsenk** [comment]
Added @lazynina 🚀  thanks!

**Comment by lazynina** [comment]
> Added @lazynina  thanks!

Please add classes to support styling the diamond popover: [see here for examples](https://github.com/bitclout/frontend/blob/main/src/styles.scss#L458)

**Comment by carsenk** [comment]
> > Added @lazynina  thanks!
> 
> Please add classes to support styling the diamond popover: [see here for examples](https://github.com/bitclout/frontend/blob/main/src/styles.scss#L458)

Not sure what you mean, the diamond popover styling is correct in all themes? Shouldn't we make a separate PR for that?

**Comment by lazynina** [comment]
> > > Added @lazynina  thanks!
> > 
> > 
> > Please add classes to support styling the diamond popover: [see here for examples](https://github.com/bitclout/frontend/blob/main/src/styles.scss#L458)
> 
> Not sure what you mean, the diamond popover styling is correct in all themes? Shouldn't we make a separate PR for that?

We pass the class when opening the popover: [see here](https://github.com/bitclout/frontend/blob/main/src/app/feed/feed-post-icon-row/feed-post-icon-row.component.ts#L467).  Without these updates, this popover will not match the theme.  The below lines should resolve it for you.

```
.diamond-popover-legends {  background-color: #14110e;  border-color: #382418;}
.diamond-popover-legends .arrow::after, .diamond-popover-legends .arrow::before {  border-top-color: #14110e;}
```

**Comment by carsenk** [comment]
> > > > Added @lazynina  thanks!
> > > 
> > > 
> > > Please add classes to support styling the diamond popover: [see here for examples](https://github.com/bitclout/frontend/blob/main/src/styles.scss#L458)
> > 
> > 
> > Not sure what you mean, the diamond popover styling is correct in all themes? Shouldn't we make a separate PR for that?
> 
> We pass the class when opening the popover: [see here](https://github.com/bitclout/frontend/blob/main/src/app/feed/feed-post-icon-row/feed-post-icon-row.component.ts#L467). Without these updates, this popover will not match the theme. The below lines should resolve it for you.
> 
> ```
> .diamond-popover-legends {  background-color: #14110e;  border-color: #382418;}
> .diamond-popover-legends .arrow::after, .diamond-popover-legends .arrow::before {  border-top-color: #14110e;}
> ```

Ohhh lol, I missed that you guys added that lol, commit incoming shortly

**Comment by lazynina** [comment]
> > > > > Added @lazynina  thanks!
> > > > 
> > > > 
> > > > Please add classes to support styling the diamond popover: [see here for examples](https://github.com/bitclout/frontend/blob/main/src/styles.scss#L458)
> > > 
> > > 
> > > Not sure what you mean, the diamond popover styling is correct in all themes? Shouldn't we make a separate PR for that?
> > 
> > 
> > We pass the class when opening the popover: [see here](https://github.com/bitclout/frontend/blob/main/src/app/feed/feed-post-icon-row/feed-post-icon-row.component.ts#L467). Without these updates, this popover will not match the theme. The below lines should resolve it for you.
> > ```
> > .diamond-popover-legends {  background-color: #14110e;  border-color: #382418;}
> > .diamond-popover-legends .arrow::after, .diamond-popover-legends .arrow::before {  border-top-color: #14110e;}
> > ```
> 
> Ohhh lol, I missed that you guys added that lol, commit incoming shortly

yep! I added it yesterday because I noticed all the diamond popovers effectively used the light theme style.

**Comment by carsenk** [comment]
Fixed and good to go! @lazynina 💯

**Review by lazynina — APPROVED** [review]

**Pull request merged**